settings/src/lib.rs

```diff
@@ -19,7 +19,8 @@ pub struct Accesso {
     pub client_id: String,
     pub client_secret: String,
     pub redirect_back_url: String,
-    pub url: String,
+    pub public_url: String,
+    pub internal_url: String,
```

Comment:
Why you need internal api url of accesso? The internal api is used by accesso’s frontend only